### PR TITLE
Beastie

### PR DESCRIPTION
**Pull request by beastie87**

### CODE REVIEW

File: app/helpers/policies_helper.rb [hunk 1]
```diff
@@ -14,9 +14,13 @@ def policy_agreement_summary_without_html(policy_member_distance)
   # Returns things like "voted strongly against", "has never voted on", etc..
   def policy_agreement_summary(policy_member_distance)
     if policy_member_distance.nil?
-      "voted <strong>unknown about</strong>".html_safe
+        "voted <strong>unknown about</strong>".html_safe
```
Comment (henare) [inline app/helpers/policies_helper.rb]: Be careful about accidentally committing stuff you don't mean to, like this superfluous whitespace.

File: app/helpers/policies_helper.rb [hunk 2]
```diff
@@ -14,9 +14,13 @@ def policy_agreement_summary_without_html(policy_member_distance)
...
     elsif policy_member_distance.number_of_votes == 0
-      "has <strong>never voted</strong> on".html_safe
+      if I18n.locale == :uk
```
Comment (henare) [inline app/helpers/policies_helper.rb:19]: Can we do this with gettext instead of an `if` exception?

File: app/helpers/policies_helper.rb [hunk 3]
```diff
@@ -14,9 +14,13 @@ def policy_agreement_summary_without_html(policy_member_distance)
...
+        "<strong>ніколи не голосував/strong>".html_safe
```
Comment (henare) [inline app/helpers/policies_helper.rb]: Typo in HTML - there's a missing `<` on the last tag.

File: app/models/policy.rb [hunk 4]
```diff
@@ -17,7 +17,7 @@ class Policy < ActiveRecord::Base
   alias_attribute :status, :private
 
   def name_with_for
-    "for #{name}"
```
Comment (henare) [inline app/models/policy.rb:20]: This model method should not be changed.

If you need the name without "for" just call `#name`.

File: app/views/help/faq.uk.html.haml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 - content_for :title, "Допомога — Часті питання"
-- set_meta_tags description: "Parliament's confusing jargon explained in plain English."
+- set_meta_tags description: 'Парламентський специфічний жаргон, пояснений простими словами.'
```
Comment (henare) [inline app/views/help/faq.uk.html.haml:2]: :+1:

File: app/helpers/policies_helper.rb (hunk 1 above)
Comment (beastie87) [inline app/helpers/policies_helper.rb]: Ok, sorry

File: app/helpers/policies_helper.rb (hunk 2 above)
Comment (beastie87) [inline app/helpers/policies_helper.rb:19]: In the English version 3, the words in Ukrainian 1 word, do not know whether or not user-friendly via gettext

**Comment by henare** [comment]
@beastie87 I've made a few comments and assigned back to you.

When you're ready for me to look at it again, just assign it to me :space_invader:

File: app/models/policy.rb (hunk 4 above)
Comment (beastie87) [inline app/models/policy.rb:20]: = capitalise_initial_character(policy.name_with_for)
replace on
= capitalise_initial_character(policy.name)

**Comment by henare** [comment]
I love the enthusiasm @beastie87 :boom: Before merging things it's a good idea to resolve all of the discussions.

It's also a good idea to not merge your own pull requests but to get someone else to so that you're sure it's been reviewed.

> Instead, if you really need to change something in the party name could we adjust the data?
> 
> I'm not clear on what we're trying to change, maybe you could clarify this?

As [I said](https://github.com/OPORA/publicwhip/pull/165#commitcomment-14863025) 656b6fe should not be merged because there's logic in the view that needs to be extracted. I'm going to revert this commit.

> Also, as I said on #158 I'd suggest not making this change anyway.

I guess my suggestion was not accepted then?

**Comment by beastie87** [comment]
I remove #gsub in views. and fix Translate "For #{some policy}" #158

**Comment by henare** [comment]
> As I said 656b6fe should not be merged because there's logic in the view that needs to be extracted. I'm going to revert this commit.

@beastie87 already reverted it in e0e4d8ecdfb2d1902b0d89cb4de61ddaf9bf0ff0 :+1:

**Comment by beastie87** [comment]
All is good?

**Comment by henare** [comment]
@beastie87 all is good :smile:
